rosbag2_to_video/api.py

```diff
@@ -30,25 +30,45 @@ def get_stamp_from_image_msg(image_msg):
 
 def add_arguments_to_parser(argparser):
     argparser.add_argument(
-        'bagfile', help='Path to the bag (currently only supports sensor_msgs/msg/Image types)')
-    argparser.add_argument('-t','--topic', required=True, help='Name of the image topic')
-    argparser.add_argument('-o','--output', required=True, help='Output filename (currently only supports MP4)')
+        'bagfile',
+        help='Path to the bag (currently only supports sensor_msgs/msg/Image or sensor_msgs/msg/CompressedImage types)' # noqa E501
+    )
+    argparser.add_argument(
+        '-t',
+        '--topic',
+        required=True,
+        help='Name of the image topic'
+    )
+    argparser.add_argument(
+        '-o',
+        '--output',
+        required=True,
+        help='Output filename (currently only supports MP4)'
+    )
     argparser.add_argument('--fps', type=int, required=True, help='Output frames per second')
+    argparser.add_argument('--storage-id', type=str, default="sqlite3", help='Rosbag2 storage id')
```

Comment:
why is this needed?
can this be read from the rosbag2 metadata?

Reply:
It is necessary if a user specify not a bag directory but a single db file as uri.

Reply:
I think requesting a directory instead of a single db is fine.
What does that happen if you specify a directory and also provide a storage id to `StorageOptions`?

If it figures it out correctly based on the metadata, then I think this change is fine.
But it would be nice to clarify that you don't need to pass the argument if you provide a directory, even if the storage id is different from `sqlite3`

Reply:
I added check if rosbag uri is a directory. https://github.com/ivanpauno/rosbag2_to_video/pull/1/commits/84034c699a47813b43db0fc1880704ef1d668de8